api/scanner/media_encoding/executable_worker/executable_worker.go

```diff
@@ -37,17 +37,17 @@ func newMagickWorker() *MagickWorker {
 		return nil
 	}
 
-	path, err := exec.LookPath("convert")
+	path, err := exec.LookPath("magick")
 	if err != nil {
-		log.Println("Executable worker not found: ImageMagick convert")
+		log.Println("Executable worker not found: magick")
```

Comment:
While the `magick` executable name better correlates with the 'ImageMagick', I'm not sure that it is intuitive enough for users to be mentioned in log messages as is...
This comment is more like a discussion starter than a change request...

Reply:
Here `magick` is the main command of ImageMagick. It follows the way of `ffmpeg` below.

With ImageMagick 6, a command like `convert` isn't able to refer ImageMagick easily, so I use `ImageMagick convert`. Now `magick` is really identifiable and no need `ImageMagick` as a prefix.